src/Admin/TimelineManifestoAdmin.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace AppBundle\Admin;
+
+use A2lix\TranslationFormBundle\Form\Type\TranslationsType;
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+class TimelineManifestoAdmin extends AbstractAdmin
+{
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
+            ->with('Traductions', ['class' => 'col-md-6'])
+                ->add('translations', TranslationsType::class, [
+                    'by_reference' => false,
+                    'label' => false,
+                    'fields' => [
+                        'title' => [
+                            'label' => 'Titre',
+                        ],
+                        'slug' => [
+                            'label' => 'URL de publication',
+                            'sonata_help' => 'Ne spécifier que la fin : http://en-marche.fr/timeline/programme/[votre-valeur]<br />Doit être unique',
+                        ],
+                        'description' => [
+                            'label' => 'Description',
+                            'filter_emojis' => true,
+                        ],
+                    ],
+                ])
+            ->end()
+        ;
+    }
+
+    protected function configureDatagridFilters(DatagridMapper $datagridMapper)
+    {
+        $datagridMapper
+            ->add('translations.title', null, [
+                'label' => 'Titre',
+                'show_filter' => true,
+            ])
+
+            ->add('translations.description', null, [
+                'label' => 'Description',
+                'show_filter' => true,
+            ])
+        ;
+    }
+
+    protected function configureListFields(ListMapper $listMapper)
+    {
+        $listMapper
+            ->addIdentifier('title', TextType::class, [
+                'label' => 'Titre',
+                'virtual_field' => true,
+                'template' => 'admin/timeline/theme/list_title.html.twig',
```

Comment:
could you try to use `translate.title` as the path of property to display and avoid using the custom template?

Reply:
doesnt work as it shows up the english translations